src/platform_linux/platform.c

```diff
@@ -7,7 +7,7 @@
 
 #include <sys/mman.h>
 
-__thread threadid xxxtid;
+__thread threadid xxxtid = INVALID_TID;
```

Comment:
Thanks for doing this initialization. It will now simplify the way I was intending to fix the issue raised in #499